doc/API/Devices.md

```diff
@@ -1367,11 +1367,11 @@ Output:
 
 Add one or more parents to a host.
 
-Route: `/api/v0/devices/:device_id/parents`
+Route: `/api/v0/devices/:device_name/parents`
```

Comment:
instead of device_name just :device is better as this more accurately potrays it allows device_id or hostname

Reply:
changed :-)